### DISCUSSION TITLE
Bulk Scan

### DISCUSSION THREAD

**Discussion by DanBenS** [Part-DB/Part-DB-server#809] [Ideas]
Hi

I hope you are doing well @jbtronics 

I would like to ask if you are interested in adding a feature which would allow the user to add/remove parts similar to a cashier system. As of now scanning multiple parts with varying quantities is not very optimal since each item would need to be scanned individually. A new "multiple Item scanner screen" would be added in order to show the contents of the bulk scan. This screen allows the user to scan the same item multiple times in order to add them or edit the quantity. Alternatively, the user can manually type the quantity (again, similar to a cashier system).

I have no problems with programming this feature. However, I would like to ask if you have any requirements which would need to be met in order for this feature to be merged with the main branch? Thanks for the help in advance

**Comment by stellardynamix** [comment]
I'd be very interested
